day-03/BinaryTree.js

```diff
@@ -0,0 +1,39 @@
+
+class Node {
+  constructor(val) {
+    this.val = val;
+    this.next = null;
+  }
+        
+}
+
+
+class BinaryTreeNode {
+  constructor(value) {
+    this.value = value;
+    this.left = null;
+    this.right = null;
+  }
+      
+  add(val) {
+    if (val < this.value) {
+      if (!this.left) {
+        this.left = new BinaryTreeNode(val);
```

Comment:
Instead of creating a new instance of your BinaryTreeNode class - you'd want to call the add method here (same goes for the findPerson challenge).